features/bicycle/destination.feature

```diff
@@ -21,15 +21,15 @@ Feature: Bike - Destination only, no passing through
             | axye  |             |
 
         When I route I should get
-            | from | to | route      |
-            | a    | b  | ab,ab      |
-            | a    | c  | ab,bcd,bcd |
-            | a    | d  | ab,bcd,bcd |
-            | a    | e  | axye,axye  |
-            | e    | d  | de,de      |
-            | e    | c  | de,bcd,bcd |
-            | e    | b  | de,bcd,bcd |
-            | e    | a  | axye,axye  |
+            | from | to | route     |
+            | a    | b  | ab,ab     |
+            | a    | c  | ab,bcd    |
```

Comment:
These changes are more or less irrelevant, since the tests themselves are disabled (marked as TODO) Apparently we do not handle destination tags for bicycles? Probably since we can walk in there anyhow in most cases.

The adjustments are only to represent what would happen if they run, since we do not announce name changes of this short a distance.